watchgod/main.py

```diff
@@ -52,12 +52,13 @@ class awatch:
     3.5 doesn't support yield in coroutines so we need all this fluff. Yawwwwn.
     """
     __slots__ = (
-        '_loop', '_path', '_watcher_cls', '_debounce', '_min_sleep', '_stop_event', '_normal_sleep', '_w', 'lock',
-        '_executor'
+        '_loop', '_path', '_watcher_cls', '_watcher_args', '_debounce', '_min_sleep', '_stop_event', '_normal_sleep',
+        '_w', 'lock', '_executor'
     )
 
     def __init__(self, path: Union[Path, str], *,
                  watcher_cls: Type[AllWatcher]=DefaultWatcher,
+                 watcher_args: Optional[Sequence]=None,
```

Comment:
Would be clearer I think to change this to `watcher_kwargs: Optional[Dict[str, Any]]`

Reply:
that also avoid the confusion that this is really "extra_watcher_args".

Reply:
How about having both `watcher_kwargs` and `extra_watcher_args`?

Reply:
I think over kill, `watcher_kwargs` should suffice in all situations.